File: yeongseong/1week/OX퀴즈.js
function solution(quiz) {
    const answer = [];
    
    for (let i = 0; i < quiz.length; i++) {
      const [x, operator, y, _, z] = quiz[i].split(" ");
      
      if (operator === "+") {
        answer[i] = parseInt(x) + parseInt(y) === parseInt(z) ? "O" : "X";
      } else if (operator === "-") {
        answer[i] = parseInt(x) - parseInt(y) === parseInt(z) ? "O" : "X";
      }
    }
    
    return answer;
  }

File: minwook/1주차/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        for (int i = 0; i < quiz.length; i++){
            String[] quizArr = quiz[i].split(" "); // 공백 기준으로 나누기
            if(quizArr[1].equals("+")){ // + 문자열 비교 (equals() 메서드 사용)
                answer[i] = Integer.parseInt(quizArr[0]) + Integer.parseInt(quizArr[2]) == Integer.parseInt(quizArr[4]) ? "O" : "X";
            }  // 참이면 O, 거짓이면 X
            else if(quizArr[1].equals("-")){
                answer[i] = Integer.parseInt(quizArr[0]) - Integer.parseInt(quizArr[2]) == Integer.parseInt(quizArr[4]) ? "O" : "X";
            }
        }
        return answer;
    }
}